c10/util/ThreadLocalDebugInfo.cpp

```diff
@@ -51,6 +51,15 @@ std::shared_ptr<DebugInfoBase> ThreadLocalDebugInfo::_pop(DebugInfoKind kind) {
   return res->info_;
 }
 
+/* static */
+std::shared_ptr<DebugInfoBase> ThreadLocalDebugInfo::_peek(DebugInfoKind kind) {
```

Comment:
Looks like the kind can be a const reference? Please ignore if it is just an integer.

Reply:
It looks like it's `uint8_t` so this should be okay